### ISSUE TITLE
Επαναυποβολή Προπαρασκευαστικής Εργασίας

### ISSUE THREAD

**Issue by kgiannakelos** [slp-ntua/patreco-lab-support-19-20#5]
Υπάρχει κάποιος τρόπος επαναυποβολής της εργασίας;

**Comment by georgepar** [comment]
Όχι. Αν έχετε κάνει κάποιο λάθος στην προπαρασκευή απλά στείλτε τη διορθωμένη μαζί με την υποβολή του πρώτου εργαστηρίου

**Comment by georgepar** [comment]
επειδή έχουν έρθει καποια mail, αν δεν είστε σίγουροι για το οτιδήποτε στην υποβολή της προπαρασκευής, στείλτε τις διορθώσεις με την πρώτη εργαστηριακη και θα ληφθούν υπόψη
